boards/arm64/xenvm/xenvm_defconfig

```diff
@@ -13,7 +13,7 @@ CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
 # Enable PV hypervisor console
-CONFIG_UART_CONSOLEIO_XEN_HVC=y
+CONFIG_UART_XEN_HVC_CONSOLEIO=y
```

Comment:
I know about that fix and it should go into another commit. It does nothing with virtual flash